docker/Dockerfile.backend

```diff
@@ -1,13 +1,14 @@
 FROM python:3.8-alpine3.10
 
-RUN apk add --no-cache --virtual .build-deps gcc libc-dev make \
+RUN apk add --no-cache --virtual .build-deps gcc libc-dev make libffi-dev \
     && pip install pipenv
 
+COPY backend/Pipfile* ./
+
 WORKDIR /app
 
-COPY backend/Pipfile Pipfile
-COPY backend/Pipfile.lock Pipfile.lock
+RUN pipenv lock --requirements > requirements.txt
 
-RUN pipenv install --dev
+RUN cat requirements.txt 
 
-EXPOSE 80
+RUN pip install -r requirements.txt
```

Comment:
If you don't like `pipenv` another popular alternative to `pipenv` is `poetry`. It seems to be much faster and many say they like it more. Haven't used it myself but if you feel like it you can play around with it.